php/class-wp-js-widget.php

```diff
@@ -100,7 +102,7 @@ public function enqueue_frontend_scripts() {}
 	 * @return array Schema.
 	 */
 	public function get_item_schema() {
-		$schema = array(
+		$item_schema = array(
 			'title' => array(
 				'description' => __( 'The title for the widget.', 'js-widgets' ),
 				'type' => array( 'string', 'object' ),
```

Comment:
And inthe case here it is allowing either a string or an object to also facilitate the REST API update requests. It means that the JSON body being sent doesn't have to supply the object but can just supply the single `title` string. Does that make sense?